src/Bonsai.ML.Visualizers/GaussianObservationsClustersVisualizer.cs

```diff
@@ -0,0 +1,332 @@
+using System;
+using System.Windows.Forms;
+using System.Collections.Generic;
+using Bonsai;
+using Bonsai.Design;
+using Bonsai.ML.Visualizers;
+using Bonsai.ML.HiddenMarkovModels.Observations;
+using OxyPlot;
+using OxyPlot.Series;
+using OxyPlot.Axes;
+using OxyPlot.WindowsForms;
+using MathNet.Numerics.LinearAlgebra;
+
+[assembly: TypeVisualizer(typeof(GaussianObservationsClustersVisualizer), Target = typeof(GaussianObservationsStatistics))]
+
+namespace Bonsai.ML.Visualizers
+{
+    /// <summary>
+    /// Provides a type visualizer of <see cref="GaussianObservationsStatistics"/> to display how the observations 
+    /// cluster with respect to the mean and covariance of each state of an HMM with gaussian observations model.
+    /// </summary>
+    public class GaussianObservationsClustersVisualizer : DialogTypeVisualizer
```

Comment:
Usually I try to name the visualizer the same as the type with `Visualizer` suffix, for ease of association. I understand in this case there is already an associated more basic visualizer, but would it make sense here to keep the naming convention and rename this as `GaussianObservationsStatisticsClusterVisualizer`?

Reply:
Updated